tdrs-backend/tdpservice/conftest.py

```diff
@@ -18,6 +19,24 @@ def user():
     return UserFactory.create()
 
 
+@pytest.fixture
+def ofa_admin():
+    """Return an ofa admin user."""
+    return UserFactory.create(groups=(Group.objects.get(name="OFA Admin"),))
+
+
+@pytest.fixture
+def ofa_analyst():
```

Comment:
@riatzukiza This role has been removed, so we should probably remove this and any tests that relate to it